### ISSUE TITLE
Read quest text.

### ISSUE THREAD

**Issue by nyaadesu** [karashiiro/TextToTalk#5]
I have an idea for an accessibility feature where this addon could read from the quest text window directly for the non voice acted quests.
Right now, there is an option to enable npc dialogue, however the text is printed to chat after skipping to next line making it hard to follow.
A hotkey could be added and it would read on press so it doesn't automatically read _everything_.
I found where the text is stored if that is of any help.
![ffxiv](https://user-images.githubusercontent.com/82879400/115387316-cda4bc80-a1da-11eb-8eef-448b17186c23.png)

**Comment by johnysandels** [comment]
Very interested to see what happens with this possible feature!

In regards to the toggle keybind request the `/toggletts` command still works well enough for turning it on and off, although I could see a keybind toggle being nice for some people, but detecting if the cutscene is voice acted would be a great alternative instead of even needing to toggle it on and off.

**Comment by karashiiro** [comment]
Sorry for the delay, been busy. Will close with https://github.com/goatcorp/DalamudPlugins/pull/843 - the only thing not implemented from this is disabling it automatically during cutscenes, which will be done with #7.

**Comment by johnysandels** [comment]
Reading the quest text works great! Although I think there should be a toggle in the TtT Config to disable it unless I overlooked the setting in the config, since some people may only want the plugin to read out FC or Tell messages etc.

**Comment by johnysandels** [comment]
Something like this would be nice.
![image](https://user-images.githubusercontent.com/79720646/118568023-eace9b80-b744-11eb-958d-4b7be356c511.png)

**Comment by karashiiro** [comment]
I'm not entirely sure I understand, but those things should be mutually exclusive, I think. To disable NPC text altogether you'd need to disable both reading from the Talk addon and the NPC Dialogue chat type. How does this look?

![Screenshot 2021-05-17 163058](https://user-images.githubusercontent.com/49822414/118568718-45f49400-b72d-11eb-8748-1c0eceb42b34.png)

**Comment by johnysandels** [comment]
Yeah just a way to disable reading NPC dialogue from the window is great. Since some people may not want to have the dialogue windows read from.

I just formatted it that way because it looks nice :)

**Comment by karashiiro** [comment]
PR merged.
